client/realm_management.go

```diff
@@ -118,8 +119,15 @@ type InstallInterfaceRequest struct {
 }
 
 // InstallInterface builds a request to install a new major version of an Interface into the Realm.
-func (c *Client) InstallInterface(realm string, interfacePayload interfaces.AstarteInterface) (AstarteRequest, error) {
+func (c *Client) InstallInterface(realm string, interfacePayload interfaces.AstarteInterface, isAsync bool) (AstarteRequest, error) {
 	callURL := makeURL(c.realmManagementURL, "/v1/%s/interfaces", realm)
+
+	query := map[string]string{}
+	if !isAsync {
+		query["async_operation"] = strconv.FormatBool(false)
+	}
+	callURL = setupURLQuery(callURL, query)
```

Comment:
I think that the if block above could become (same for UpdateInterface):
```suggestion
	if !isAsync {
	        query := map[string]string{"async_operation":  strconv.FormatBool(false)}
		callURL = setupURLQuery(callURL, query)
	}
```

Reply:
Updated